executor/distsql.go

```diff
@@ -1212,7 +1212,9 @@ func (w *tableWorker) executeTask(ctx context.Context, task *lookupTableTask) er
 				obtainedHandlesMap.Set(handle, true)
 			}
 
-			logutil.Logger(ctx).Error("inconsistent index handles", zap.String("index", w.idxLookup.index.Name.O),
+			logutil.Logger(ctx).Error("inconsistent index handles",
+				zap.String("table_name", task.partitionTable.Meta().Name.O),
```

Comment:
partitionTable is used only for the partition table. It's not the same as the table.

Reply:
fixed